src/redisai.c

```diff
@@ -805,12 +805,71 @@ int RedisAI_ScriptScan_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **arg
     return REDISMODULE_OK;
 }
 
+void _RedisAI_Info(RedisModuleCtx *ctx) {
+    RedisModuleString *rai_version = RedisModule_CreateStringPrintf(
+        ctx, "%d.%d.%d", REDISAI_VERSION_MAJOR, REDISAI_VERSION_MINOR, REDISAI_VERSION_PATCH);
+    RedisModuleString *llapi_version =
```

Comment:
Should we update our LLAPI version after supporting async and DAG? currently it is defined as "1".